core/integration/settings.py

```diff
@@ -139,6 +149,11 @@ class ConfigurationFormItemType(Enum):
     SELECT = "select"
     LIST = "list"
     MENU = "menu"
+    NUMBER = "number"
```

Comment:
I'll change the usage to `PositiveInt` types but will keep the `ConfigurationFormItemType.NUMBER` as the `type` since this changes the Admin UIs behaviour to only allow numbers in the text field it generates.